src/aws_sagemaker/aws_sagemaker.py

```diff
@@ -221,6 +201,53 @@ async def get_object_point_clouds(self,
                                       **kwargs) -> List[PointCloudObject]:
         raise NotImplementedError
     
+    async def get_properties(
+            self,
+            *,
+            extra: Optional[Mapping[str, Any]] = None,
+            timeout: Optional[float] = None) -> Vision.Properties:
+                return Vision.Properties(
+                classifications_supported=True,
+                detections_supported=True,
+                object_point_clouds_supported=False,
```

Comment:
This depends on the specific AWS model though, doesn't it? You would have check what the user is doing

Reply:
There's no way to check what the user is loading (short of trying it and seeing if it fails).  I could try something like that on reconfigure (or on GetProps call) and then save the properties, but I figured it makes more sense to say that they both are supported, which is generally true, and trust whoever is using the aws-sagemaker model to know whether they have a classifier or detector.